README.md

```diff
@@ -41,3 +41,23 @@ n = 10**5
 sleep_durations = [random.uniform(0, 5.0) for _ in range(n)]
 asyncio.run(run_multiple_sleeps(sleep_durations=sleep_durations))
 ```
+
+## Demo
+A complete demo of `tqdm_publisher` can be found in the `demo` directory, which shows how to forward progress updates from the same `TQDMPublisher` instance to multiple clients.
+
+To run the demo, first install the dependencies:
+```bash
+pip install -r demo/requirements.txt
+```
+
+Then, run the Python script for the demo:
+```bash
+python demo/main.py
+```
+
+Finally, open the demo web page in your browser:
+```bash
+open demo/index.html
```

Comment:
Likewise, might want to make a simple CLI exposure to make this call without having to know where the HTML file is